topics/transcriptomics/tutorials/scrna-preprocessing-tenx/tutorial.md

```diff
@@ -0,0 +1,449 @@
+>                nAmbigFeature           9537
+>        nAmbigFeatureMultimap           7473
+>                     nTooMany            691
+>                nNoExactMatch           1750
+>                  nExactMatch         106272
+>                       nMatch         106868
+>                nCellBarcodes            272
+>                        nUMIs          52748
+> ```
+{: .comment}
+
+The explanation of these parameters can be seen in the [RNA STAR Manual](https://github.com/alexdobin/STAR/blob/master/doc/STARmanual.pdf) under the *STARsolo* section, with most of the information given in the details box below.
+
+> ### {% icon details %} Explanation of parameters
+>
+>  | Parameter | Explanation |
+>  |-----------|-------------|
+>  | nNinBarcode | number of reads with more than 2 Ns in cell barcode (CB) |
+>  | nUMIhomopolymer | number of reads with homopolymer in CB |
+>  | nTooMany | not used at the moment |
+>  | nNoMatch | number of reads with CBs that do not match whitelist even with one mismatch |
+>
+> All of the above reads are discarded from Solo output.
+>
+> The remaining reads are checked for any overlap with the features/genes:
+>
+>
+>  | Parameter | Explanation |
+>  |-----------|-------------|
+>  | nUnmapped | number of reads unmapped to the genome |
+>  | nNoFeature | number of reads that map to the genome but do not belong to a feature |
+>  | nAmbigFeature | number of reads that belong to more than one feature |
+>  | nAmbigFeatureMultimap | number of reads that belong to more than one feature and are also multimapping to the genome (this is a subset of the `nAmbigFeature`) |
+>  | nTooMany | number of reads with ambiguous CB (i.e. CB matches whitelist with one mismatch but with posterior probability 0.95) |
+>  | nNoExactMatch | number of reads with CB that matches a whitelist barcode with 1 mismatch, but this whitelist barcode does not get any other reads with exact matches of CB |
+>
+> These metrics can be grouped into more broad categories:
+>
+> * `nNinBarcode` + `nUMIhomopolymer` + `nNoMatch` + `nTooMany` + `nNoExactMatch` = number of reads with CBs that do not match whitelist.
+> * `nUnmapped` + `nAmbigFeature` = number of reads without defined feature (gene)
+> * `nMatch` = number of reads that are output as solo counts
+>
+> The three categoties above summed together should be equal to the total number of reads.
+>
+{: .details}
+
+The main information to gather at this stage is that the `nCellBarcodes` tell us how many cells were detected in our sample, where we see 272 which is expected of our sub-sampled data. Another metric to take into account is that the number of matches (`nMatch`) has the largest value, and that the number of reads that map to the genome but not to a feature/gene given in the GTF (`nNoFeature`) is not too large. The number of no features is also quite high when mapping the source files, so this appears to be the default expected behaviour.
```

Comment:
the term "source files" is confusing to me, can you call them original files from the paper?